swarm/pss/prox_test.go

```diff
@@ -102,7 +103,12 @@ func getCmdParams(t *testing.T) (int, int) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	return int(msgCount), int(nodeCount)
+	timeoutStr := fmt.Sprintf("%ss", args[3])
```

Comment:
This does not seem as canonical Go to me. As you append the unit to the string before parsing that into duration. In Go I have to provide the amount and the unit of time with duration flags, e.g., `go test -timeout 5s`. (`go test -timeout 5` would fail)

Reply:
I don't see why this matters. But feel free to provide a concrete suggestion and I can change it.

Reply:
My comments superseded by https://github.com/ethersphere/go-ethereum/pull/1304#issuecomment-474833132